internal/telemetry/round_stats.go

```diff
@@ -0,0 +1,160 @@
+package telemetry
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/arkade-os/arkd/internal/core/application"
+	otelLog "go.opentelemetry.io/otel/log"
+	"go.opentelemetry.io/otel/log/global"
+	"time"
+)
+
+type RoundReportLogExporter struct {
+	cancel context.CancelFunc
+	done   chan struct{}
```

Comment:
seems to be used only in Close, we can get rid of it and just use the cancel function to stop listening to the svc.Report channel.